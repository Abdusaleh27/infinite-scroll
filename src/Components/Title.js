import React from "react";

const Title = () => {
  return (
    <nav className="navbar sticky-top">
      <div className="container-fluid">
        <h4 className="text-light mx-auto">Infinite Scroll</h4>
      </div>
    </nav>
  );
};

export default Title;
