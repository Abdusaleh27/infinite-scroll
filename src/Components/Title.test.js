import Title from "./Title";
import { render,screen } from "@testing-library/react";

//smoke test
test("Title renders",() => {
    render(<Title />);
});

//snapshot text
test("Title snapshot",() => {
    const {asFragment} = render(<Title />);
    expect(asFragment()).toMatchSnapshot();
})

//contains the intended title
it("contains the title 'Infinite Scroll'",() =>{
    render(<Title />);
    expect(screen.getByText("Infinite Scroll")).toBeInTheDocument();
});