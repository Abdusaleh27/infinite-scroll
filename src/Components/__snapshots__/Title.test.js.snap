// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title snapshot 1`] = `
<DocumentFragment>
  <nav
    class="navbar sticky-top"
  >
    <div
      class="container-fluid"
    >
      <h4
        class="text-light mx-auto"
      >
        Infinite Scroll
      </h4>
    </div>
  </nav>
</DocumentFragment>
`;
